### ISSUE TITLE
Module not found: Error: Package path ./dist/index is not exported from package /dev/project/node_modules/prosemirror-proofread (see exports field in /dev/project/node_modules/prosemirror-proofread/package.json)

### ISSUE THREAD

**Issue by fridaystreet** [nullpointerexceptionkek/prosemirror-proofread#3]
Struggling to get this to work as per the readme. Am I missing something here?

Module not found: Error: Package path ./dist/index is not exported from package /dev/project/node_modules/prosemirror-proofread (see exports field in /dev/project/node_modules/prosemirror-proofread/package.json)

```js
import { Extension } from '@tiptap/core'
import {
  createProofreadPlugin,
  createSpellCheckEnabledStore,
} from 'prosemirror-proofread'
import { generateProofreadErrors, createSuggestionBox } from './utils'

const spellCheckEnabledStore = createSpellCheckEnabledStore(true);
const proofreadPlugin = createProofreadPlugin(
  1000, // Debounce time in ms
  generateProofreadErrors as never, // function to call proofreading service
  createSuggestionBox, // Suggestion box function
  spellCheckEnabledStore, // Reactive store to toggle spell checking
);

export const SpellCheck = Extension.create({
  name: 'spellcheck',
  addProseMirrorPlugins() {
    return [proofreadPlugin];
  }
})
```

If I try importing default as oer the readme, it just says default not exported. It doesn't complain when I try and import as above, but when I try and build with reatc webpack, I get the error

**Comment by nullpointerexceptionkek** [comment]
I'll take a look once I get home.

**Comment by nullpointerexceptionkek** [comment]
I will need some time to fix these issues,
If you need an quicker fix, just copy this into your project folder
src/lib/proofreadPlugin.ts

**Comment by nullpointerexceptionkek** [comment]
The issue should be solved in 0.2.2, please see the react example here
https://github.com/nullpointerexceptionkek/prosemirror-proofread/blob/master/demos/reacttest/src/App.tsx

**Comment by fridaystreet** [comment]
all good thanks, yep just reverted to copying the lib out into my project and ran it directly.

Nice implmentation btw
